README.md

```diff
@@ -30,3 +30,5 @@ In chronological order:
 [@azyobuzin](https://github.com/azyobuzin) -- providing code for finding places where RC.EXE could hide
 
 [@retep998](https://github.com/retep998) -- fixing MSVC support
+
+[@application-developer-da](https://github.com/application-developer-da) -- adding support for Windows crosscompilation (GNU toolkit)
```

Comment:
Ok, do you want me to remove all these mentions from both `README.md` and `Cargo.toml`?